.travis.yml

```diff
@@ -3,11 +3,11 @@ python:
  - "2.7"
  - "3.3"
 env:
- - DJANGO_VERSION=1.8.17
- - DJANGO_VERSION=1.9.12
- - DJANGO_VERSION=1.10.6
+ - DJANGO_VERSION_MIN=1.8 DJANGO_VERSION_MAX=1.9
+ - DJANGO_VERSION_MIN=1.9 DJANGO_VERSION_MAX=1.10
+ - DJANGO_VERSION_MIN=1.10 DJANGO_VERSION_MAX=1.11
 install:
- - "pip install django==$DJANGO_VERSION"
+ - "pip install django=>$DJANGO_VERSION_MIN,<$DJANGO_VERSION_MAX"
```

Comment:
I didn't know `=>` was a valid operator. Should that be: `...django>=$DJANGO...`